packages/story-editor/src/karma/fixture/db/getTaxonomiesResponse.js

```diff
@@ -285,7 +285,7 @@ export default [
       ],
       'wp:items': [
         {
-          href: 'http://localhost:8899/wp-json/wp/v2/story-verticals',
+          href: 'http://localhost:8899/wp-json/web-stories/v1/web_story_category',
```

Comment:
there were a lot of rebases here, @littlemilkstudio is this the right href here?

Reply:
`web-stories/v1/web_story_category` is correct